CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+## 0.26.0 / 2022-08-07
```

Comment:
is it rc.0 or the final release?

Reply:
Fixed in https://github.com/prometheus/alertmanager/pull/3447/commits/24b74165ceeb8a07c4b8a0e115d2d296e9330bdb